src/domain/auth/authenticate.ts

```diff
@@ -11,9 +14,22 @@ import { TUNNISTAMO_API_TOKEN_ENDPOINT } from '../api/constants/ApiConstants';
 import { BackendTokenResponse } from './types/BackendAuthenticationTypes';
 
 export const loginTunnistamo = (path?: string) => {
-  userManager.signinRedirect(
-    path ? { data: { path: path } } : { data: { path: '/profile' } }
-  );
+  userManager
+    .signinRedirect(
+      path ? { data: { path: path } } : { data: { path: '/profile' } }
+    )
+    .catch(error => {
+      if (error.message === 'Network Error') {
+        toast(i18n.t('authentication.networkError.message'), {
```

Comment:
it could be better if you wrote an util
displayErrorNotification= (arg) => toast(...args, { type: toast.TYPE.ERROR })

Reply:
And use it every where instead of repeating that error toast

Reply:
Good idea, I will make an issue for this.